### ISSUE TITLE
Dark dialog message background issue in UI

### ISSUE THREAD

**Issue by rohityadavcloud** [apache/cloudstack#9643]
Dark/black background seen in alert/dialog messages:

<img width="577" alt="Screenshot 2024-09-05 at 3 14 32 PM" src="https://github.com/user-attachments/assets/dabf4b58-2057-4362-bfec-46da085058ec">

This was from a npm run serve env (not a built env), with following `npm version` output:
```
{
  npm: '8.19.4',
  node: '16.20.2',
  v8: '9.4.146.26-node.26',
  uv: '1.48.0',
  zlib: '1.2.12',
  brotli: '1.1.0',
  ares: '1.28.1',
  modules: '93',
  nghttp2: '1.61.0',
  napi: '8',
  llhttp: '6.0.11',
  openssl: '3.2.1',
  cldr: '44.1',
  icu: '74.2',
  tz: '2023c',
  unicode: '15.1'
}
```

##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
main/4.2.0
~~~

**Comment by rohityadavcloud** [comment]
This is still often seen - we don't know exactly what causes this :( cc @Pearl1594 @DaanHoogland

**Comment by DaanHoogland** [comment]
yes, Rohit, I think it was there before 4.20 as well. Cache clear sometimes seems to work

**Comment by weizhouapache** [comment]
> [@rohityadavcloud](https://github.com/rohityadavcloud) Do you think [#10497](https://github.com/apache/cloudstack/pull/10497) will solve this issue as well?

@Pearl1594 
it seems so

I am closing this issue. If the same issue appears, let's reopen it
